11/chapter-12.srt

```diff
@@ -54,7 +54,7 @@ to do that, but we'll get to that in a second.
 
 13
 00:01:20,620 --> 00:01:25,040
-"undefined method `>' for nil:NilClass
+"undefined method `>' for nil:NilClass"
```

Comment:
I just though if you were going to start the quotes, they should be finished? Possibly not needed at all. Just a thought.